<?xml version="1.0"?>
<launch>
    <node name="local_server_node" pkg="mqtt_cast" type="mqtt_listener.py"  output="screen" respawn="false">
        <!-- set private parameters for the node -->
        <param name="mqtt_debug"  type="bool" value="true" />
        <param name="broker_ip"  type="str" value="172.16.0.94" />
    </node>

</launch>